### DISCUSSION TITLE
how to login with multiple table

### DISCUSSION THREAD

**Discussion by MNhat0308** [filamentphp/filament#10838] [Help]
First of all thanks creator of this package
like the title , i have another table store user that can login, so how i can reuse login route filament for login on both table
default user for redirect filament route 
another table redirect to another route which is not filament
Thanks!

**Comment by leandrocfe** [comment]
Could you explain more about it please?

**Reply by MNhat0308** [reply]
i have another the table call member, i want use both table member and user for login
but when login with table member and want redirect to another route not admin route filament
